exonum/src/sandbox/consensus/transactions.rs

```diff
@@ -311,16 +317,12 @@ fn incorrect_tx_in_request() {
 
 #[test]
 fn response_size_larger_than_max_message_len() {
-    use crypto::{PUBLIC_KEY_LENGTH, SIGNATURE_LENGTH};
-    use messages::HEADER_LENGTH;
+    use messages::TRANSACTION_RESPONSE_EMPTY_SIZE;
     use storage::StorageValue;
 
-    const EMPTY_RESPONSE_SIZE: usize =
-        (HEADER_LENGTH + SIGNATURE_LENGTH + 2 * PUBLIC_KEY_LENGTH + 8);
-
     let sandbox = timestamping_sandbox();
     let sandbox_state = SandboxState::new();
-
+    const TX_HEADER: usize = 8 + 2;
```

Comment:
I'd put this constant before local variables and it could use some explanation of its value.